config/frontend.example.toml

```diff
@@ -31,6 +31,10 @@ timeout = "0s"
 ## The following units are supported: `B`, `KB`, `KiB`, `MB`, `MiB`, `GB`, `GiB`, `TB`, `TiB`, `PB`, `PiB`.
 ## Set to 0 to disable limit.
 body_limit = "64MB"
+## Maximum total memory for all concurrent HTTP request bodies.
+## Set to 0 to disable the limit. Default: "0" (unlimited)
```

Comment:
better set a reasonable default, like 100MB, for better out-of-the-box user experience

Reply:
Providing a default number is challenging right now because the difference between protocol compression and uncompression can be substantial. For now, we must rely on user decisions; users can monitor metrics to determine the appropriate value for their system.